integration/tests/git_source/git_source_repeated_remote.rb

```diff
@@ -0,0 +1,65 @@
+require 'git_utils'
+require 'r10k_utils'
+require 'master_manipulator'
+test_name 'Verify the same remote can be used in more than one object'
+
+env_path = on(master, puppet('config print environmentpath')).stdout.rstrip
+r10k_fqp = get_r10k_fqp(master)
+git_environments_path = '/root/environments'
+git_repo_path = '/git_repos'
+git_repo_name = 'environments'
```

Comment:
this is.. confusing.. can we call this something else, like 'control-repo' or sometthing?

Reply:
or is this just so that we don't have to configure (what file-sync calls) a submodules dir?

Reply:
This corresponds to the control repo, hosted locally instead of somewhere like Github. We could call it something else, but this is the name used throughout in these tests, so I'm a little reluctant to change it just here.

Reply:
right. I'm fine leaving it if that's what we do elsewhere, I feel like it just conflates things in a way that is kind of confusing. but I assume that's it's being used so that paths look the way we expect them to, with an `environments` directory.

Reply:
The name of this path shouldn't have any bearing on where things get placed, since r10k is just treating this directory/git repo like a remote. It's likely called `environments` because ultimately a control repo is just the collection of environments, each of its branches corresponding to a different one. Likely this helped the original test writer feel less confused about what was happening, even if we've gotten super used to other names (like "control repo") now.

Reply:
This is what the control-repo fixture is called throughout these tests (an I think maybe in frankenbuilder too) so I left it.

Reply:
hmm. then I like it even less lol. but it's fine, we can leave it for consistency.

Reply:
The `env_path` var above is what holds the location that content will be deployed to, assuming it doesn't have its `install_dir` overridden.

Reply:
ahh okay. so then for our purposes, we don't need to set `install_dir` at all, right?

Reply:
Nope. And I'm changing that, because I'm updating this to install the same module under two different names, but both in the regular modulepath.